### PR TITLE
refactor: destroy tippy only for outdated elements

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

https://app.clickup.com/t/1t7hjx5

The issue described on the ticket is because the current implementation removes all the tippy instances (and creates them again) every time livewire updates the DOM.

The was intended to fix an issue with tippy tooltips that never disappeared when an element disappears from the DOM while have the cursor over keeping the tooltip forever.

This PR refactors the implementation to check for every previous tippy instance and remove the ones that are no longer on the DOM

### To test

- Merge the changes from this PR on nodem
- The issue described on the ticket should not happen anymore
- Also you need ways to emulate an element disappearing on the DOM while the cursor is over an element or find places when you can see that issue. I personally hardcoded conditions to randomly hide a tooltip and then tested manually.
- This PR affects every place where we have tippy implementations so you may need to think in potential problems with my new implementation

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [x] I provided a screenshot of my changes to the component _(if applicable)_
-   [x] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [x] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Review by ItsANameToo — APPROVED** [review]

**Review by alexbarnsley — APPROVED** [review]
General comment for a possible refactor but otherwise works and nothing wrong with what you've done :+1:

File: resources/assets/js/tippy.js [hunk 1]
```diff
@@ -41,10 +53,34 @@ const destroyTippy = (parentEl = document.body) => {
...
+                el.getAttribute("data-tippy-content"))
+        ) {
+            filteredTippyInstances.push(instance);
+        } else {
+            instance.destroy();
+        }
+    });
+
+    tippyInstances = filteredTippyInstances;
+};
```
[the remark this answers is not in the document]
Comment (alfonsobries) [reply resources/assets/js/tippy.js:82]: @alexbarnsley seems ok but ill do manual testing

[the remark this answers is not in the document]
Comment (alfonsobries) [reply resources/assets/js/tippy.js:80]: @alexbarnsley yep it worked, thanks

**Review by leMaur — APPROVED** [review]
Go fly 🚀

**Pull request merged**